src/api/include/migraphx/migraphx.hpp

```diff
@@ -1343,6 +1343,14 @@ struct onnx_options : MIGRAPHX_HANDLE_BASE(onnx_options)
     {
         call(&migraphx_onnx_options_set_limit_loop_iterations, this->get_handle_ptr(), value);
     }
+
+    /// Set absolute path for external data files
+    void set_external_data_path(const std::string& external_data_path)
```

Comment:
Can you add a simple test example with external data path ? 
for example something like this 
https://github.com/ROCm/AMDMIGraphX/blob/2c4dd4a9e4af590539ddee7e021b99443cff9cda/test/api/test_cpu.cpp#L104 ?? 

we have onnx simple onnx file with ext data here which you can make use of or else you can generate a new one on your own. 

https://github.com/ROCm/AMDMIGraphX/tree/develop/test/onnx/ext_path

Reply:
Done.

Reply:
Can you add `compile` and `eval` similar to this one ? 
https://github.com/ROCm/AMDMIGraphX/blob/2c4dd4a9e4af590539ddee7e021b99443cff9cda/test/api/test_cpu.cpp#L110

Reply:
I added the changes.